plugins/azure/resoto_plugin_azure/resource/network.py

```diff
@@ -5008,8 +4966,8 @@ def connect_in_graph(self, builder: GraphBuilder, source: Json) -> None:
             path="/subscriptions/{subscriptionId}/providers/Microsoft.Network/networkInterfaces",
             path_parameters=["subscriptionId"],
             query_parameters=["api-version"],
-            compared_property=compared_property,
-            binding_property=binding_property,
+            first_property=first_property,
+            second_property=second_property,
         )
```

Comment:
This call is not required - it fetches all network interfaces, which have been already fetched and are available in the graph.